src/main/java/com/sda/auction/controllers/AuctionController.java

```diff
@@ -1,14 +1,44 @@
 package com.sda.auction.controllers;
 
+import com.sda.auction.entities.Auction;
+import com.sda.auction.entities.User;
 import com.sda.auction.services.AuctionService;
 import lombok.AllArgsConstructor;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Optional;
 
 @RestController
 @AllArgsConstructor(onConstructor = @__({@Autowired}))
 public class AuctionController {
 
     private final AuctionService auctionService;
 
+    @GetMapping("/auctions")
+    public List<Auction> findAllAuctions(){
+        return auctionService.findAllAuctions();
+    }
+
+    @GetMapping("/auction/{id}")
```

Comment:
We frequently use the word "auction". Let's make a constant in the class and use that constant